docs/UsersGuide/source/BackgroundInfo/CCPPUpdates.rst

```diff
@@ -0,0 +1,95 @@
+* Bug fixes for GFS-based thermal roughness length scheme
+* New soil color dataset introduced to improve soil albedo to reduce the large warm bias found in the Sahel desert
+* Wet leaf contribution factor is included
+* Leaf-area index now depends on momentum roughness length
+
+
+RUC Land Surface Model
+========================
+
+* Initialization of land and ice emissivity and albedo with consideration of partial snow cover
+* Initialization of water vapor mixing ratio over land ice
+* Initialization of fractions of soil and vegetation types in a grid cell
+* Changes in the computation of a flag for sea ice: set to true only if ``flag_cice=.false`` (atmosphere uncoupled from the sea ice model).
+* Separate variables for sea ice, for example: ``snowfallac`` is replaced with ``snowfallac_ice``
+* Solar angle dependence of albedo for snow-free land
+* Stochastic physics perturbations (SPP) introduced for emissivity, albedo and vegetation fraction
+* Coefficient in soil resistance formulation (Sakaguchi and Zeng, 2009) raised from 0.7 to 1.0 to increase soil resistance to evaporation
+* Computation of snow cover fraction and snow thermal conductivity updated
+
+GFS Scale-Aware TKE-EDMF PBL and Cumulus Schemes
+==================================================
+
+* Parameterization to represent environmental wind shear effect added to reduce excessively high hurricane intensity
+* Entrainment rates enhanced proportionally to the sub-cloud or PBL-mean TKE when TKE is larger than a threshold value
+* Entrainment rate is increased as a function of vegetation fraction and surface roughness length to enhance underestimated CAPE
+
+MYNN-EDMF PBL Scheme
+======================
+
+* Small increase of buoyancy length scale in convective environments
+* Patch for ensuring non-zero cloud fractions for all grid cells where cloud mixing ratio is greater than 1e-6 or ice mixing ratio is greater than 1e-9
+
+Subgrid-Scale (SGS) Clouds Scheme
+===================================
+
+* Bug fix for cloud condensate input into RRTMG radiation
+* New code section for use with SAS convection scheme
+* Cloud fraction now computed as a mix between the area-dependent form and the modified Chaboureau and Bechtold (2005) form
+* Adjusted limit for the boundary flux functions
+
+MYNN Surface-layer Scheme
+===========================
+
+* Reintroduced friction velocity averaging over water to reduce noise in 10-m winds in the hurricane regime
+
+Grell-Freitas Scale and Aerosol Aware Convection Scheme
+=========================================================
+
+* Update for aerosol-awareness (experimental)
+* Scale-awareness turned off when explicit microphysics is not active anywhere in the column
+* Convection is completely suppressed at grid points where the MYNN PBL scheme produces shallow convection
+* Radar reflectivity considers mass flux PDF as well as whether scale-awareness is turned on at the grid point in equation
```

Comment:
`at the grid point in equation` --> `at the grid point in *question*`?

Reply:
@mkavulich Can you clarify this?

Reply:
@mkavulich Looking for an update on this point, too.

Reply:
It looks like this change references https://github.com/ufs-community/ccpp-physics/pull/64. In this change, it is indeed a point-wise scale-awareness check, and dx can technically be different from gridpoint to gridpoint. Realistically this would only every happen if you were right around the threshold that they've set here (6.5 km), since the ESG grid has very uniform dx for most cases.

Reply:
@JeffBeck-NOAA Does this answer your question? If so, could you approve the PR? Or suggest a rewording in light of Mike's clarification?

Reply:
@gspetro-NOAA @mkavulich So after reading Mike's comment, it sounds like this change should be made:

`at the grid point in equation --> at the grid point in *question*`

Otherwise I don't understand what that sentence means.

Reply:
@gsketefian I changed it.